carma_guidance_plugins/src/control_plugin.cpp

```diff
@@ -74,10 +74,15 @@ namespace carma_guidance_plugins
         [this]() {
           if (this->get_activation_status()) // Only trigger when activated
           {
-            this->vehicle_cmd_pub_->publish(this->generate_command());
+            auto vehicle_cmd = this->generate_command();
+            //If vehicle cmd is outdated - dont publish
+            if ((this->now() - rclcpp::Time(vehicle_cmd.header.stamp)).seconds() < 5.0){
```

Comment:
Let's make this 5.0 value a named member variable (likely `cmd_timeout_in_sec` or similar) in the `ControlPlugin` class to avoid the use of a magic number here

Reply:
Resolving this comment since we are removing and updates related to the vehicle command timeout from this PR.